task/build-vm-image/0.1/build-vm-image.yaml

```diff
@@ -65,7 +65,7 @@ spec:
         name: varlibcontainers
   steps:
     - name: use-trusted-artifact
-      image: quay.io/redhat-appstudio/build-trusted-artifacts:latest@sha256:0379f5c2feec921a01300b386ce9414b10acfbc4499b187700d972b2360e1585
+      image: quay.io/redhat-appstudio/build-trusted-artifacts:latest@sha256:11d90835f4cc882715c37f16513db19876f346096ccba0f26d548b13bfd4a270
```

Comment:
For some reason this one has a different digest from all the others :thinking:

Reply:
Ah, it's the amd64 manifest while the others are the multi-arch image index. That's probably fine

Reply:
Good catch, I think we should change it to the `c91de1...`, for consistency and it'll be less confusing with that, I'll push a commit on this branch